resources/config/zype.config.php

```diff
@@ -5,4 +5,7 @@
  */
 
 global $zype_wp_options;
+
+$zype_wp_options['auth_url'] = 'signin';
```

Comment:
@Osoro Could you please remove the bits for the `auth_url` fix (we've already fixed that in another commit) and rebase against the current master?
Thank you!